snippets/saved_replies.md

```diff
@@ -0,0 +1,105 @@
+This document is useful for copy and pasting into https://github.com/settings/replies - feel free to chnage wording to your preference.
+It's good to have a bit of variety in our responses, anyway.
+If you are are using more than one response at the same time, cutting out the initial thank you and replacing it with "Also, " should be enough. Any combination should create a sane flow, even if most obviously wouldn't make much sense.
+
+All these snippets follow the same basic layout
+
+1. Thank people.
+They actually took the time to create an issue/PR. Some took more time, some less time; for some it was hard, for others it was trivial. Disregarding all this, we simply thank the first as they are doing more than 99% or our userbase.
+2. Acknowledge their need for help.
+Unless it's a PR for a typo, most likely they actually feel a need for whatever they are asking about.
+2. Give context.
+Considerations, better places to go with this, etc. If it's anything we have docs on, enable them to come up to speed with the factual basis we have.
+3. Provide a course of action.
+At the least, if we close anything quickly, we should tell them they can ask to re-open, or where else to take this. Again, take into account that there's a need they feel they need to address, somehow.
+
+It's not always feasible, or even possible, to fill out the XXX. Still, we should try to.
+
+
+
+# Issues
+
+
+## User Support
+
+Thanks for your report. It looks as if this is actually a question about usage and not development.
+
+To make your question, and all replies, easier to find, we suggest you move this over to our [user mailing list](https://groups.google.com/forum/#!forum/prometheus-users), which you can also search. If you prefer more interactive help, join or our [IRC channel](https://webchat.freenode.org/?channels=prometheus), #prometheus on irc.freenode.net. Please be aware that our IRC channel has no logs, is not searchable, and that people might not answer quickly if they are busy or asleep. If in doubt, you should choose the mailing list.
+
+If you think this is not purely a support question, feel free to comment in here or take the underlying issues to our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers).
+
+Once your questions have been answered, please add a short line pointing to relevant replies in case anyone stumbles here via a search engine in the future.
+
+All that being said, in your specific case, it might make sense to / please look at / our reasoning can be found at XXX
+
+
+# PRs
+
+
+## DCO
+
+Thanks for your PR. For legal reasons, we require that all commits are signed with a [DCO](https://developercertificate.org/) before we can merge them.
```

Comment:
Is there a link to the specific legal reasons or an explanation (beyond the DCO page) or does that cover it?

Reply:
The LF, and thus the CNCF requires it. @caniszczyk, do you have a link handy?

Reply:
it's in the CNCF Charter's IP Policy: https://www.cncf.io/about/charter/

If you want to know why... here's some links:
https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/
https://blog.chef.io/2016/09/19/introducing-developer-certificate-of-origin/
https://blog.docker.com/2014/01/docker-code-contributions-require-developer-certificate-of-origin/